homeassistant/components/ring/binary_sensor.py

```diff
@@ -75,35 +78,23 @@ class RingBinarySensor(RingEntityMixin, BinarySensorEntity):
 
     def __init__(
```

Comment:
Have you checked if you can inherit from the CoordinatorEntity here? That would be the preferred way for entities that are based on a DataUpdateCoordinator. See https://github.com/home-assistant/core/blob/e594c19c1ecb9bc947b37d2af75e8d84f6a922e9/homeassistant/components/overkiz/entity.py#L18 for an example.

Reply:
This would make your code also cleaner since you don't have to implement `_handle_coordinator_update` everywhere, but you can just focus on your entity logic.

Reply:
Resolving conversation as I've clarified that we are inheriting from `CoordinatorEntity`.